client/test/grammars/snaps/bitbake-conf.conf

```diff
@@ -0,0 +1,20 @@
+BBPATH .= ":${LAYERDIR}"
+
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
+	${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "swupdate"
+
+BBFILE_PATTERN_swupdate := "^${LAYERDIR}/"
```

Comment:
Since this code is obiously copied from meta-swupdate, we should add a comment stating the original license and source. However, you can rename swupdate here. The arrangment is an original work I guess.